src/olympia/core/db/base.py

```diff
@@ -0,0 +1,49 @@
+from django.db.backends.mysql.base import (
+    CursorWrapper as MySQLCursorWrapper,
+    DatabaseWrapper as MySQLDBWrapper,
+    DatabaseIntrospection as MySQLDBIntrospection,
+    DatabaseSchemaEditor as MySQLDBSchemeEditor)
+
+
+class CursorWrapper(MySQLCursorWrapper):
+    def execute(self, query, args=None):
+        return super(CursorWrapper, self).execute(query, args)
```

Comment:
Actually, class isn't necessary - I was debugging using it to verify the correct sql was being generated.  I will drop it along with `DatabaseWrapper.create_cursor` and `DatabaseSchemaEditor.column_sql` methods.